src/controllers/followController.js

```diff
@@ -0,0 +1,206 @@
+import Models from '../db/models';
+import errorResponse from '../helpers/index';
+
+const { User, Follow } = Models;
+
+/**
+ * @description handles follow and unfollow
+ * @class FollowController
+ */
+class FollowController {
+  /**
+   * @description Handles the follow feature
+   * @param {object} request - Request object
+   * @param {object} response - Response object
+   * @returns {object} An object containing all the data related to the unfollowed user
+   * @memberof FollowController
+   */
+  static async followUser(request, response) {
+    const { username } = request.params;
+    const followingId = request.userData.payload.id;
```

Comment:
Nice Implementation Frosty 👍 , I love how you used the JWT payload data to validate in this method